frontend/public/components/dashboard/project-dashboard/details-card.tsx

```diff
@@ -44,6 +46,11 @@ export const DetailsCard: React.FC = () => {
               <GreenCheckCircleIcon /> Service Mesh Enabled
             </DetailItem>
           )}
+          {description && (
+            <DetailItem isLoading={!obj} title="Description">
+              <span className="co-line-clamp details-card__description">{description}</span>
```

Comment:
With the CSS change above this can be

```suggestion
              <span className="details-card__description">{description}</span>
```